miner/worker_test.go

```diff
@@ -783,8 +778,9 @@ func testCommitInterruptExperimentBorContract(t *testing.T, delay uint, txCount
 	w.stop()
 
 	currentBlockNumber := w.current.header.Number.Uint64()
-	assert.Check(t, txCount >= w.chain.GetBlockByNumber(currentBlockNumber-1).Transactions().Len())
-	assert.Check(t, 0 < w.chain.GetBlockByNumber(currentBlockNumber-1).Transactions().Len()+1)
+	prevBlockTxCount := w.chain.GetBlockByNumber(currentBlockNumber - 1).Transactions().Len()
+	assert.Check(t, prevBlockTxCount > 0)
```

Comment:
> `w.chain.GetBlockByNumber(currentBlockNumber-1).Transactions().Len()+1`

Not sure why it was incremented by `1` before, but the new logic makes more sense.